pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [tool.black]
 line-length = 100
 skip-string-normalization = true
-target_version = ['py37', 'py38']
+target_version = ['py310']
```

Comment:
@ametzger is this what you meant/the correct approach? I got rid of the py37/py38 values and set it to be py310 only

Reply:
Yup, perfect, thanks!